.github/workflows/build-node-bridge.yml

```diff
@@ -0,0 +1,62 @@
+name: "[Build] suite-node-bridge-bin-js"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  push:
+    branches:
+      - develop
+      - grdddj/ci_node_bridge_build
+  pull_request:
+    types: [labeled]
+  workflow_dispatch:
+
+jobs:
+  node-bridge-bin-js:
+    name: Build node-bridge bin.js file
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+          submodules: true
+
+      - name: Install node and yarn
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Install deps and build libs
+        run: |
+          yarn install --immutable
```

Comment:
is this full yarn install needed? wouldn't `yarn focus` to transport-bridge folder suffice?

Reply:
No idea at all here, but will try it

Reply:
this way it installs all the dependencies. In frequently run jobs, it makes sense to optimize it using yarn focus. here it is less relevant but still a good practice to follow if possible